samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/LayerLists.razor

```diff
@@ -62,4 +85,20 @@
         return Task.Run(() => item);
     }
 
+    private SceneView? _baseview;
```

Comment:
I'm realizing that neither of the view refs are actually ever called, so they could be deleted.

Reply:
removed in both layerlists and new basemaplayerlists